fearless/Modules/Staking/SelectedValidators/SelectedValidatorsViewController.swift

```diff
@@ -68,7 +83,7 @@ extension SelectedValidatorsViewController: UITableViewDelegate, UITableViewData
             )!
 
         let items = viewModel?.itemViewModels ?? []
```

Comment:
and it could crash at line 86(
```swift
let row = indexPath.row
guard let items = viewModel?.itemViewModels, row < items.count else { return }
let item = items[row]
...
```

Reply:
Well, in this case looks like it should crash because it would explicitly show data inconsistency